pandas/core/groupby/generic.py

```diff
@@ -851,6 +851,10 @@ def fillna(
             Method to use for filling holes. ``'ffill'`` will propagate
             the last valid observation forward within a group.
             ``'bfill'`` will use next valid observation to fill the gap.
+
+            .. deprecated:: 2.1.0
+                Use obj.ffill or obj.bfill instead.
```

Comment:
Since we're going to deprecate the method argument, we should just deprecate all of `fillna` here; using `value` with groupby provides no benefit over using it in e.g. `DataFrame.fillna`. I was already leaning this way while I was working on #53438.

Reply:
Good point, will update.

Reply:
would you be OK with just doing the method deprecation here and a separate PR to deprecate the rest of the method?

Reply:
Yea - sounds good.